contracts/LiquidDemocracy.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity ^0.4.4;
+
+contract LiquidDemocracy {
+    ////////////////
+    // Data structures
+    ////////////////
+
+    // This is the type for a single option shown in a ballot
+    struct Proposal {
+        bytes32 name; // short name (up to 32 bytes)
+        uint voteCount; // number of accumulated votes
+    }
+
+    // A dynamically-sized array of `Proposal` structs.
+    Proposal[] public proposals;
+
+    // This is the type for a single voter metadata
+    struct Voter {
+        uint weight;
+        bool voted;
+        // bool delegated;
+        bool registered;
+    }
+    
+    mapping(address => Voter) votersRegistration;
```

Comment:
I would prefer the e.g. votersData name. There seem to be / potentially will be information here unrelated to the registration (weight, voted, etc)

Reply:
Fair enough. Naming is something we should strive to get right; the easier to understand, the better